src/frontend/src/common/api/index.tsx

```diff
@@ -3,7 +3,7 @@ import { jsonToType } from "src/common/utils";
 
 export enum API {
   USER_INFO = "/api/usergroup",
-  SAMPLES = "/api/samples",
+  SAMPLES = "/v2/samples/",
```

Comment:
I think we changed the response format -- there's more work to do before the frontend can just switch over to the new endpoint.

Reply:
changed back to old endpoint!